### PR TITLE
DOC-2773-CG/NG-transition

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Added info about the CG/NG transition button

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md [hunk 1]
```diff
@@ -106,9 +107,30 @@ For details, go to [Policy as Code](/docs/category/policy-as-code).
...
+If this setting is set to true, you can switch between CD FirstGen and NextGen with the click of a button. This feature is currently behind the feature flag, `PLG_ENABLE_CROSS_GENERATION_ACCESS`.
+
+The default `allowCrossGenerationAccess` value for NextGen is `TRUE`. 
+
+The default `allowCrossGenerationAccess` value for FirstGen is:
+
+* `TRUE` if the feature flag, `PL_HIDE_LAUNCH_NEXTGEN` is disabled.  
+* `FALSE` if the feature flag, `PL_HIDE_LAUNCH_NEXTGEN` is enabled.
+
+For new FirstGen and NextGen user accounts, this value is set to `FLASE` by default.
```
Comment (kapil-harness) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md:121]: there is a typo, make FLASE -> FALSE

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md [hunk 2]
```diff
@@ -106,9 +107,30 @@ For details, go to [Policy as Code](/docs/category/policy-as-code).
 
 ## Upgrading to NextGen CD
 
-Please reach out to your Customer Success Manager (CSM), Account Executive, or your Customer Success Engineer (CSE) for help upgrade to Harness NextGen CD. 
+An account setting, `allowCrossGenerationAccess` is introduced to enable cross generational access between CD FirstGen and NextGen. This setting is visible to CD subscribed (free/paid account) users only. This setting can be changed by account admins only. 
+
+If this setting is set to true, you can switch between CD FirstGen and NextGen with the click of a button. This feature is currently behind the feature flag, `PLG_ENABLE_CROSS_GENERATION_ACCESS`.
+
+The default `allowCrossGenerationAccess` value for NextGen is `TRUE`.
```
Comment (kapil-harness) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md:114]: users can get confused, what is NextGen?
we should mentioned:
The default `allowCrossGenerationAccess` value for accounts with defaultExeperience as NextGen is `TRUE`.

**Review by kapil-harness — APPROVED** [review]

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md [hunk 3]
```diff
@@ -106,9 +107,30 @@ For details, go to [Policy as Code](/docs/category/policy-as-code).
 
 ## Upgrading to NextGen CD
 
-Please reach out to your Customer Success Manager (CSM), Account Executive, or your Customer Success Engineer (CSE) for help upgrade to Harness NextGen CD. 
+An account setting, `allowCrossGenerationAccess` is introduced to enable cross generational access between CD FirstGen and NextGen. This setting is visible to CD subscribed (free/paid account) users only. This setting can be changed by account admins only.
```
Comment (aimurphy) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md:110]:
```suggestion
Added an account setting, `allowCrossGenerationAccess`, that you can use to enable cross generational access between CD FirstGen and NextGen. This setting is visible to CD subscribed (free/paid account) users only. This setting can be changed by account admins only. 
```

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md [hunk 4]
```diff
@@ -106,9 +107,30 @@ For details, go to [Policy as Code](/docs/category/policy-as-code).
 
 ## Upgrading to NextGen CD
 
-Please reach out to your Customer Success Manager (CSM), Account Executive, or your Customer Success Engineer (CSE) for help upgrade to Harness NextGen CD. 
+An account setting, `allowCrossGenerationAccess` is introduced to enable cross generational access between CD FirstGen and NextGen. This setting is visible to CD subscribed (free/paid account) users only. This setting can be changed by account admins only. 
+
+If this setting is set to true, you can switch between CD FirstGen and NextGen with the click of a button. This feature is currently behind the feature flag, `PLG_ENABLE_CROSS_GENERATION_ACCESS`.
```
Comment (aimurphy) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md:112]:
```suggestion
If this setting is set to true, you can switch between CD FirstGen and NextGen with the click of a button. This feature is currently behind the feature flag, `PLG_ENABLE_CROSS_GENERATION_ACCESS`.
```

Where is the button located?

Suggestion: `...you can switch between CD FirstGen and NextGen by selecting **Launch Harness Next/First Generation** on the **Page**.`

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md [hunk 5]
```diff
@@ -106,9 +107,30 @@ For details, go to [Policy as Code](/docs/category/policy-as-code).
...
+| `FALSE` | UI will not show the **Launch Harness Next Generation** button | UI will not show the **Launch First Generation** button |
 
-For any further questions or comments, please contact your assigned CSE or CSM. We appreciate your business and look forward to continuing to serve your business needs in the future.
+To upgrade to NextGen, select the **Launch Harness Next Generation** button in the Harness FirstGen UI. 
 
+![](./static/launch-harness-next-gen.png)
 
+To go back to FirstGen, select the **Launch First Generation** button in the Harness NextGen UI.
 
+Reach out to your assigned Customer Success Manager (CSM), Account Executive, or your Customer Success Engineer (CSE) for help upgrade to Harness NextGen, or for any further queries or comments.
```
Comment (aimurphy) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md:136]:
```suggestion
Reach out to your assigned Customer Success Manager (CSM), Account Executive, or your Customer Success Engineer (CSE) for help upgrading to Harness NextGen, or for any further queries or comments. 
```

**Review by aimurphy — APPROVED** [review]
approved w some comments

**Pull request merged**